source/kubernetes/index.html.md

```diff
@@ -1,31 +1,59 @@
 ---
-title: GOV.UK Kubernetes cluster documentation for developers
+title: GOV.UK Internal Developer Platform
 weight: 1
 layout: multipage_layout
 ---
 
-# GOV.UK Kubernetes cluster documentation for developers
+# GOV.UK Internal Platform
 
-Use this documentation to find out how to:
+The GOV.UK Internal Platform is a centralised platform where teams can build, deploy, and manage applications in a reliable and secure environment.
 
-- start using the GOV.UK Kubernetes clusters
-- manage your GOV.UK app on Kubernetes
+This documentation is for both platform users and internal team members. It provides guides, tools, and best practices to help onboard, manage, and troubleshoot applications efficiently.
 
-## Introduction
+## User Guide
 
-The GOV.UK website and content management system (known internally as the publishing platform) are hosted on [Kubernetes](https://kubernetes.io/) clusters on Amazon Web Services (AWS), using [Elastic Kubernetes Service](https://aws.amazon.com/eks/) (EKS).
+This section is for application developers using the platform. The platform intends to reduce cognitive load by abstracting infrastructure complexity and providing tools, guides, and automated workflows.
 
-Kubernetes is a widely-used, open-source [container orchestration system](https://cloud.google.com/discover/what-is-container-orchestration) for managing containerised workloads such as GOV.UK's.
+If something is missing, let us know, and we’ll add an article. If you're comfortable contributing, pull requests (PRs) are welcome.
```

Comment:
Add an anchor link to the #contact-the-platform-engineering-team section? Or direct to the page? (And possibly in the next paragraph too)